cmd2/argparse_completer.py

```diff
@@ -407,9 +407,15 @@ def update_mutex_groups(arg_action: argparse.Action) -> None:
                                 if action.dest != argparse.SUPPRESS:
                                     parent_tokens[action.dest] = [token]
 
-                                completer = ArgparseCompleter(
-                                    self._subcommand_action.choices[token], self._cmd2_app, parent_tokens=parent_tokens
-                                )
+                                parser: argparse.ArgumentParser = self._subcommand_action.choices[token]
+                                completer_type: Optional[
+                                    Type[ArgparseCompleter]
+                                ] = parser.get_ap_completer_type()  # type: ignore[attr-defined]
```

Comment:
Since we have to do these few lines everywhere we use this, would it make more sense to change `get_ap_completer_type()` to return `DEFAULT_AP_COMPLETER` if nothing has been set?

Reply:
I made that code common in a cmd2 method. I prefer letting `get_ap_completer_type()` still return `None` and let the caller determine how to handle it.